### DISCUSSION TITLE
Support for LilyGO T-Echo

### DISCUSSION THREAD

**Discussion by fab1an2** [markqvist/Reticulum#214] [Help & Questions] [answered]
Is possible using this?

**Comment by Erethon** [comment, marked as answer]
It seems to be using the `SX1262` LoRa transceiver, which according to the [Rnode documentation](https://github.com/markqvist/RNode_Firmware/blob/ee110738645310df56497bde13f1b53456c636a5/README.md#supported-transceiver-modules) support for it is being implemented, but isn't there yet.

**Comment by markqvist** [comment]
@fab1an2, please don't use the issue tracker to ask questions. Post them in the forum. Every time I have to move stuff like this around, it takes 15 minutes longer to get SX1262 support in the RNode Firmware ;)

**Comment by fab1an2** [comment]
Maybe instead of building our own few devices, ally with pine64 and together create a thousend device that is compatible with our network.

They have the capacity, a large number of engineers and a huge (by open source standards) number of volunteers.

https://pine64.com/product-category/internet-of-things/iot-lora/

**Reply by markqvist** [reply]
This is the whole point here already. To idea with RNode is to create a firmware and software systems that works across practically all available transceiver types. It already supports hundreds of thousands of devices available all over the world. There's thousands of them in operation.

There's no point in "allying" with **one** specific hardware vendor at this point. The hardware is already there.

If you really want your own hardware, you can also just use the RNode platform to design from, of course. Then you can build exactly the kind of device you want.

**Reply by fab1an2** [reply]
>The hardware is already there.
 then why is no one using it?

In my opinion, the hardware not exist. 'allyying' is for create alternative firmware for this device pine64. 
or creating together next level device. Probably Pine64 fundation not undestand trouble radio propagations etc.
They announced that this equipment would already work years ago and nothing is happening. The software is beyond them and they don't understand how the radio works. It's not as simple as they thought. That is why he writes about allying with them.

**Reply by markqvist** [reply]
Honestly, I've lost you here. I have no idea what you are actually talking about. Sorry ;)
